src/workloads/scale/TimeSeriesSortScale.yml

```diff
@@ -0,0 +1,554 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+    This test exercises the behavior of the time series bounded sorter as the number of documents
+    in the collection increases.
```

Comment:
I think it would helpful to add a couple more points here:
* The collection has events from many separate time-series in it: the metadata can take on 1000 different values.
* On each insert phase we insert more events, but we don't increase the number of series.
* We expect the bounded sort to scale linearly, and the blocking sort to scale worse than linearly.